store/postgres/src/subgraph_store.rs

```diff
@@ -772,12 +776,13 @@ impl SubgraphStoreTrait for SubgraphStore {
 
         let graft_base = match store.graft_pending(id)? {
             Some((base_id, base_ptr)) => {
-                let site = self.primary_conn()?.find_existing_site(&base_id)?;
-                Some((Arc::new(site), base_ptr))
+                let src = self.layout(&base_id)?;
+                Some((src, base_ptr))
             }
             None => None,
         };
-        store.start_subgraph(logger, site, graft_base)
+        store.start_subgraph(logger, site.clone(), graft_base)?;
+        self.primary_conn()?.copy_finished(site.as_ref())
```

Comment:
It reads a bit weird to execute this unconditionally even though most subgraphs won't be copied when deployed, maybe this `copy_finished` call can be moved into `start_subgraph`?

Reply:
Unfortunately, this can't go into `start_subgraph` since it manipulates a different database.